cmd/gke-gcloud-auth-plugin/cred/cred.go

```diff
@@ -77,9 +77,23 @@ func newPlugin(tokenProvider tokenProvider) *plugin {
 	}
 }
 
+// TokenProviderType represents which type of tokens to retrieve
+type TokenProviderType int
+
+const (
+	// Gcloud type provides gcloud OAth 2.0 tokens
+	Gcloud             TokenProviderType = 0
```

Comment:
```suggestion
Gcloud TokenProviderType = iota
DefaultCredentials
GcloudEdgeCloud
```

This makes it easy to just add on more.

Also, the spacing here seems a little wonky - is this what the linter wants?

Reply:
Done, thanks